### PR TITLE
Introduce upgrade recovery script

### PR DESCRIPTION
**Pull request by donpenney**
This update also adds the go-bindata package to allow generation of go
code that includes the upgrade recovery script.

Signed-off-by: Don Penney <dpenney@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ spec:
         volumeMounts:
         - name: container-backup
           mountPath: /host
+      hostNetwork: true
```
Comment (sabbir-47) [inline README.md:32]: You could add backoffLimit to 0 with this PR

**Comment by danielmellado** [comment]
@donpenney just a quick comment, do we need for the script to go on private folder? I understand that that's something you don't expect to be imported outside of this, but nevertheless.

File: README.md (hunk 1 above)
Comment (donpenney) [reply README.md:32]: Yes, I could. If I have a need to update this PR, I will include that.

**Comment by donpenney** [comment]
> 

The generated code can be placed anywhere. Given that it's not something we'd have a reason to export to an external module, it seemed "internal" was appropriate.

File: README.md (hunk 1 above)
Comment (donpenney) [reply README.md:33]: Done

**Review by danielmellado — APPROVED** [review]
/lgtm

**Pull request merged**